packages/nuxt/src/core/nitro.ts

```diff
@@ -213,7 +213,7 @@ export async function initNitro (nuxt: Nuxt & { _nitro?: Nitro }) {
   // Add app manifest handler and prerender configuration
   if (nuxt.options.experimental.appManifest) {
     // @ts-expect-error untyped nuxt property
-    const buildId = nuxt.options.appConfig.nuxt!.buildId ||= randomUUID()
+    const buildId = nuxt.options.appConfig.nuxt!.buildId ||= (nuxt.options.test ? 'test' : randomUUID())
```

Comment:
Haven't checked impl but is `dev` also considered?

Reply:
Do you mean in dev (non-test) we should default to `dev` or something similar?

Reply:
Yes (either this field is optional or if we can serve it dev, i think a predictable value could be nice).